src/models/settings.js

```diff
@@ -0,0 +1,66 @@
+const mongoose = require("mongoose");
+const mongodbErrorHandler = require("mongoose-mongodb-errors");
+const findOrCreate = require("mongoose-findorcreate");
+
+// Subschema for the Facebook authentication provider credentials.
+const FacebookAuthProviderCredentialsSchema = new mongoose.Schema({});
+
```

Comment:
I thought you defined these so that you will use them in the `settingsSchema`.

Reply:
I ended up making them nested paths as the IDs generated for each subdocument was unneccessary.

Reply:
Just forgot to remove the variables for those schemas.